Contributing guide update and Community guide (#796)
nephio-project/nephio: COMMUNITY.md

# Nephio Project Community membership

This document describes the responsibilities and contributor roles in Nephio Project. 

| Role | Responsibilities | Requirements | Defined by |
| -----| ---------------- | ------------ | -------|
| New contributor |  - | Submits the Pull Request on GitHub | Anyone is welcome to participate |
| Member | Active contributor | Sponsored by one reviewer and has several contributions to the project | Nephio GitHub org member|
| Reviewer | Review contributions from other members | History of review and authorship in the project | [OWNERS] file reviewer entry |
| Approver | Contributions acceptance approval| Highly experienced active reviewer and contributor to the project | [OWNERS] file approver entry|

## New contributor

Anyone is welcome to contribute to Nephio project. Existing members should make a point of being helpful 
for new contributors in any way possible (PR workflow, finding relevant documentation, guiding with coding style etc.)

> Remember we were all '[new contributors]' once.

Reviewer: ```suggestion
Remember we were all *new contributors* once.
```

## Member

Members are active contributors in the community. Members might have PRs and GitHub issues assigned to them and are added to GitHub team(s).
Prow's presubmit tests are automatically run for their PRs as well as GitHub Actions (where applicable). 
Members are welcome to remain continuously active contributors to the community.

### Requirements

> - Ensure affiliation is up to date in [openprofile.dev]. 

Reviewer: Should this be a link? "[openprofile.dev]"

Author: Links are defined at the bottom of doc: https://github.com/radoslawc/nephio/blob/2f99de696abb492386973cdbd9441799d85c9385/COMMUNITY.md?plain=1#L107

- Have made **several contributions** to the project or community, enough to

>   demonstrate an **ongoing and possible long-term commitment** to the project like:

Reviewer: ```suggestion
  demonstrate an **ongoing and possible long-term commitment** to the project such as:
```

    - Authoring or reviewing PRs on GitHub, with at least one **merged** PR.
    - Filing or commenting on issues on GitHub
    - Contributing to SIG, subproject, or community discussions (meetings,
      Slack, mailing list)

> - Have read the [contributor guide]

Reviewer: Should '[contributor guide]" be a link?

Author: You're right! Fixed

- Sponsored by one reviewer

> - **Open an issue [Membership request] against the 'nephio' repo**

Reviewer: ```suggestion
- **Open an issue **Membership request** against the 'nephio' repo**
```

   - Ensure your sponsor is @mentioned on the issue
   - Provide information described above that is representative of your work on the project
- Have your sponsoring reviewer reply confirmation of sponsorship
- Once your sponsors have responded, your request will be reviewed by the [GitHub Admin team].

### Responsibilities and privileges

- Responsive to issues and PRs assigned to them
- Responsive to mentions of SIG teams they are members of
- Active owner of code they have contributed
  - Code is well tested
  - Tests consistently pass
  - Addresses bugs or issues discovered after code is accepted

> - Members can do `/lgtm` on all PRs.

Reviewer: I don't think this is the case now.

Author: LGTM? As far as I know should be on every repository, 'approve' is more strict.
